### PR TITLE
Update lukarukikato portfolio website for assignment task

### PR DESCRIPTION
**Pull request by lukarukikato**
Updating the readme.md file by adding the personal website of academician Muhammad Favian Revaldy [FAV}. Website will be further improved in a later date.

### CODE REVIEW

**Review by fakhrip — COMMENTED** [review]
Wow thats a very long url you got there, it looks like a wrong url to me as it seems to include a generated hash from vercel, no?

**Comment by lukarukikato** [comment]
> Wow thats a very long url you got there, it looks like a wrong url to me as it seems to include a generated hash from vercel, no?

Ah apologies for the mistake, the URL should be fixed now.

File: Readme.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ Welcome all of you future star candidates who want to strive to attend the acade
   > As an academician.  
   > Website link -> [https://portofolio-radif.vercel.app/](https://portofolio-radif.vercel.app/)
 - [Muhammad Favian Revaldy](https://github.com/lukarukikato) [FAV]
-  > As an academician.
+   > As an academician.
```
Comment (fakhrip) [inline Readme.md:73]: Next time you should check if the whitespacing is correct or not, but its fine for now since github could still render it properly

**Review by fakhrip — APPROVED** [review]
Good job!

File: Readme.md (hunk 1 above)
Comment (lukarukikato) [reply Readme.md:73]: My apologies again, it must've slipped by. I'll be sure to be more careful next time.

**Pull request merged**